features/lib/step_definitions/language_steps.rb

```diff
@@ -6,3 +6,7 @@
     expect(all_output.force_encoding('utf-8')).to include(language)
   end
 end
+
+Then(/^the output includes the message "(.*)"$/) do |message|
```

Comment:
This is a generic step that doesn't belong in 'language steps' - better to put it in `aruba_steps.rb` with the similar steps.